src/components/tasks/card/card.module.scss

```diff
@@ -5,6 +5,22 @@ $yellow: #ecef08;
 $white: #e5e5e5;
 $black: #252525;
 
+.srOnly{ //screen reader only
```

Comment:
Instead of comment, would it be better to name the className with similar name?

Reply:
okay

Reply:
done